bundles/org.openhab.binding.avmfritz/src/main/java/org/openhab/binding/avmfritz/internal/handler/AVMFritzBaseBridgeHandler.java

```diff
@@ -270,11 +269,10 @@ public void onDeviceListAdded(List<AVMFritzBaseModel> deviceList) {
                 .collect(Collectors.toMap(it -> it.getIdentifier(), Function.identity()));
         getThing().getThings().forEach(childThing -> {
             final AVMFritzBaseThingHandler childHandler = (AVMFritzBaseThingHandler) childThing.getHandler();
+            // if (childHandler != null && ThingHandlerHelper.isHandlerInitialized(childHandler)) {
```

Comment:
Is this line expected to be here?

Reply:
No, that was my backup plan ... removed.